src/config/backends/raspberry-pi.ts

```diff
@@ -0,0 +1,157 @@
+import * as _ from 'lodash';
+import { fs } from 'mz';
```

Comment:
This should be able to be
```suggestion
import { promises as fs } from 'fs';
```
now that the supervisor uses nodejs 12 and work the same, except using the in-built promise interface (it was added in nodejs 10)
- the same goes for other usage as well

Reply:
I've created https://github.com/balena-io/balena-supervisor/issues/1384 to address this in another PR.